src/Control/Optics/Linear/Internal.hs

```diff
@@ -48,7 +48,7 @@ import Data.Void
 import GHC.Exts (FUN)
 import GHC.Types
 import Prelude.Linear
-import qualified Prelude as P
+import qualified Prelude as Ur
```

Comment:
This probably are not in scope of this PR, but I was wondering if we should have some consistency here. I think I saw all three:

```haskell
import qualified Prelude
import qualified Prelude as NonLinear
import qualified Prelude as Ur
```

It might be good to decide on one. (My favourite is the first one, I guess, since we don't want single letter imports).

Reply:
`Ur` is two letters one. Would you be OK with that?

Reply:
My problem is not about it being two letters, but more about we already having a `Data.Unrestricted.Linear` module and a `Ur` data type. So, when I see an `Ur` qualifier in a codebase, I am more likely to think of `Data.Unrestricted.Linear` instead of `Prelude`.

Reply:
Very fair ... I'll go with `import qualified Prelude` then.

Reply:
There would be nothing wrong with never creating an alias for modules when importing them, except that it would be unbearably verbose. Because at least that way, when seeing an occurrence of the module in source code, readers know exactly what module it is. So shortening long hierarchical module names while still retaining the most significant bit of their name (usually the final component) strikes the right balance.